modules/configuring-assumerole-without-iam.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-with-image-registries.adoc
+:_module-type: PROCEDURE
+[id="configuring-assumerole-without-iam_{context}"]
+= Configuring AssumeRole without container IAM
+
+[role="_abstract"]
+If you want to use AssumeRole without container IAM, you must use an an access and a secret key to authenticate as an link:https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users_create.html[AWS user with programmatic access].
+
+.Procedure
+. If the user for which you want to assume role is in the same account as the ECR registry, create a new role with the desired permissions.
++
+[NOTE]
+====
+When creating the role, you can choose any trusted entity as required. However, you must modify it after creation.
+====
++
+Otherwise, if the user is in a different account than the ECR registry, you must assign the role to a policy that allows the role to assume another role as required:
```

Comment:
I don't think the role is required to have the permissions to assume another role, the role in the account with the registry only needs the permissions desired to access the ECR registry, and to have it's trust relationship defined

Reply:
It's the user which needs the permission to assume role provided to it